string data_filename
string text_filename
---
bool success
